### DISCUSSION TITLE
How to remove a release of ingress-nginx-controller from artifactub.io

### DISCUSSION THREAD

**Discussion by longwuyuan** [artifacthub/hub#2952] [Q&A] [answered]
- Please look at this issue https://github.com/kubernetes/ingress-nginx/issues/9824
- The ingress-nginx project would benefit if it was possible to remove a specific release from artifacthub.io
- Is there a documentation or a contact that could help in removing a specific release from artifacthub

cc @tao12345666333  @strongjz @rikatz

**Comment by tegioz** [comment, marked as answer]
Hi @longwuyuan 👋 

If you remove that version from the [Helm's repo index.yaml file](https://kubernetes.github.io/ingress-nginx/index.yaml) (it's still there), Artifact Hub should delete it as well 🙂

**Reply by longwuyuan** [reply]
@tegioz thanks. I think the Helm repo itself is updated using the github release process so I am not aware how to modify index.yaml there. 

@strongjz @rikatz @tao12345666333 please advise

**Reply by tegioz** [reply]
No worries! Removing the [entry corresponding to that version in the index.yaml file](https://github.com/kubernetes/ingress-nginx/blob/5911b0aabfe16c333b3bb7ef4d524674ae47e098/index.yaml#L42-L79) should do the trick.

**Comment by longwuyuan** [comment]
Ah ok. So a unconventional edit to the index.yaml in the root of our
project is the task. Clear now. Thanks tons.

**Comment by strongjz** [comment]
fixed in https://github.com/kubernetes/ingress-nginx/pull/9827
